src/Orbit.js

```diff
@@ -55,9 +67,14 @@ export function getOrbitType(ephem) {
  */
 export class Orbit {
   /**
-   * @param {Ephem} ephem The ephemerides that define this orbit.
+   * @param {Object} ephem The ephemerides that define this orbit.
```

Comment:
This should still be `Ephem` type, or the type below on line 85 needs to be updated.

Reply:
Well it's overloaded now so it's either an Ephem or EphemerisTable. Does documentation support {Ephem | EphemerisTable}?

Reply:
Yeah, I think the proper jsdoc notation would be `{(Ephem | EphemerisTable)}`